protoc-cli/src/main/resources/google/protobuf/wrappers.proto

```diff
@@ -0,0 +1,117 @@
+// Protocol Buffers - Google's data interchange format
+// Copyright 2008 Google Inc.  All rights reserved.
+// https://developers.google.com/protocol-buffers/
+//
+// Redistribution and use in source and binary forms, with or without
```

Comment:
`ballerina/protobuf/descriptor.proto` is missing right (resource file)?

Reply:
Added in [6d4d80b](https://github.com/ballerina-platform/protoc-tools/pull/3/commits/6d4d80b05617ed62f2910e14b60e0d1dc4177cca)